liblwgeom/liblwgeom.h.in

```diff
@@ -34,14 +34,24 @@
 #include <stdint.h>
 
 #include "../postgis_config.h"
-#include "proj_api.h"
 
-#if POSTGIS_PROJ_VERSION >= 49
-/* Enable new geodesic functions */
-#define PROJ_GEODESIC 1
+#if POSTGIS_PROJ_VERSION < 60
```

Comment:
You are changing the style of this file from `tabs` to 4 spaces :oncoming_police_car: My bet is `.h.in` isn't detected by your editor as a C header.

Reply:
Editorconfig has `h.in` set to spaces... not saying this is right, just saying it's what the file seems to be

Reply:
I'd never checked the contents of editorconfig. I'm not sure if I would dare to do it again.